modules/sagemaker/sagemaker-model-monitoring/README.md

```diff
@@ -68,15 +74,24 @@ N/A
 
 ###### Required
 
-- `ground-truth-prefix`: The S3 prefix in `model-artifacts-bucket-arn` which contains the [ground truth](https://docs.aws.amazon.com/sagemaker/latest/dg/model-monitor-model-quality-merge.html) for captured data.
 - `model-quality-problem-type`: The machine learning problem type of the model that the monitoring job monitors.
-- `model-quality-inference-attribute`: The attribute of the input data that represents the ground truth label.
 
 ###### Optional
 
+- `model-quality-inference-attribute`: The attribute of the input data that represents the ground truth label.
 - `model-quality-probability-attribute`: In a classification problem, the attribute that represents the class probability.
 - `model-quality-probability-threshold-attribute`: The threshold for the class probability to be evaluated as a positive result.
 
+##### Model Bias Monitoring Job Parameters
+
+###### Optional
+
+- `model-bias-checkstep-analysis-config-prefix`: The S3 prefix in `model-artifacts-bucket-arn` which contains the output from the [Clarify Check Step](https://docs.aws.amazon.com/sagemaker/latest/dg/build-and-manage-steps.html#step-type-clarify-check) used for model bias.
+- `model-bias-features-attribute`: The attributes of the input data that are the input features.
+- `model-bias-inference-attribute`: The attribute of the input data that represents the ground truth label.
+- `model-bias-probability-attribute`: In a classification problem, the attribute that represents the class probability.
```

Comment:
I need to check if it makes sense for these to be separate parameters from the model quality job.

Reply:
I think we'll just go with this for now to leave it flexible.